RestCommClient/Classes/RCCallKitProvider.h

```diff
@@ -38,24 +37,31 @@
 - (void)callEnded;
 @end
 
+/**
+ *  @abstract RCCallKitProvider handles the CallKit logic.
+ *  @param connection The RCConnection instance
```

Comment:
Is this param for the constructor below instead of the interface? If so still let's keep the description for the interface

Reply:
Fixed. It should be pushed it..